astrovascpy/bloodflow.py

```diff
@@ -632,12 +639,13 @@ def construct_static_incidence_matrix(graph):
     return sparse.csc_matrix((data, (row, col)), shape=(graph.n_edges, graph.n_nodes))
 
 
-def _solve_linear(laplacian, input_flow):
+def _solve_linear(laplacian, input_flow, params=None):
```

Comment:
```suggestion
def _solve_linear(laplacian, input_flow, params={}):
```
This is in conjunction with the suggestion later

Reply:
No mutable data structures as Python default arguments? I would not do that!

Reply:
the changes to params does not need to affect the outside. In this case it is totally safe